### ISSUE TITLE
[question] Are std::ranges::advance, std::ranges::next ... customisation points?

### ISSUE THREAD

**Issue by h-2** [ericniebler/range-v3#1058]
The old documentation for the TS suggests this:
https://en.cppreference.com/w/cpp/experimental/ranges/iterator/advance

But the current wording of the DS does not seem to:
http://eel.is/c++draft/iterators#range.iter.ops

Did I miss something in the draft standard or is there a reason this was dropped?

Thanks!

**Comment by ericniebler** [comment]
They are not customization points, and they never were, even in the TS. (`advance` is a customization point in range-v3 for the time being, but I've already changed that in the v1.0-beta branch.)

They were specified to be "customization point object," which is a fancy term of art for global function object. Its a misnomer since there is no way to customize it's behavior (it never tries to call `advance` unqualified, for example) so we changed how it was specified to about the term.
